stingray/events.py

```diff
@@ -171,10 +171,10 @@ def simulate_times(self, lc, use_spline=False, bin_time=None):
         events_per_bin_predict = n_events_predict / n_bin
         
         if use_spline:
-            max_bin = np.max([4, 1000000 / events_per_bin_predict])
+            max_bin = np.long(np.max([4, 1000000 / events_per_bin_predict]))
```

Comment:
This is because of the latest Numpy requirement for integer type when specifying array indices or slices (`a[1.]` was tolerated before, not anymore). I used `np.long` because it's a clearly defined type and I don't think we'll have soon a time series with a number that can overflow it, but I might use Python's `int` to be as general as possible.